File: FOURTH/CLA/sum.java
import java.*;

class sum
{
    public static void main(String argv[])
    {
        if(argv.length!=2)
        {
            System.out.println("Insufficeint Parameters");
        }
        else
        {
            int a=Integer.parseInt(argv[0]);
            int b=Integer.parseInt(argv[1]);
            int c=a+b;
            System.out.println("Sum = "+c);
        }
    }
}

File: FOURTH/CLA/sum.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc,char *argv[])
{
    int a,b,c;
    if(argc!=3)
    {
        printf("Insufficeint Parameters");
    }
    else
    {
        a=atoi(argv[1]);
        b=atoi(argv[2]);
        c=a+b;
        printf("Sum = %d\n",c);
    }
    return 0;
}